code/game/turfs/open/floor/mineral_floor.dm

```diff
@@ -119,6 +119,39 @@
 /turf/open/floor/mineral/titanium/purple/airless
 	initial_gas_mix = AIRLESS_ATMOS
 
+// OLD TITANIUM (titanium floor tiles before PR #50454)
+/turf/open/floor/mineral/titanium/old
+	name = "titanium tile"
+	icon_state = "titanium_old"
```

Comment:
why the `old` part in the name?

Reply:
It's called "old" because it's the old sprites returning back into the game.
It's just called that because I can't think of a better name that lets mappers/coders know what it is better than "it's the old titanium floor sprite".

Reply:
They're called "tiled" now.
![image](https://user-images.githubusercontent.com/53862927/95148278-d7a12980-077a-11eb-803e-759490c0aaba.png)
It's kinda... tile-y?

Reply:
Can you also change the `icon_state` names to match?